index.md

```diff
@@ -113,6 +136,50 @@ Just because we're in a pub doesn't mean you have to drink alcohol.  If you do c
 
 Although the venue for this part of the meeting is a pub and therefore feels more informal, be aware that the code of conduct still applies.
 
+## Mailing list
+
+The mailing-list can be found at [lrug.org/mailing-list](http://lrug.org/mailing-list).  From here you will be able to sign up, manage your account, and view the archives.  The traffic is quite variable; on some days there'll be no messages, other days there'll be several independent threads with many active participants.
+
+All messages to the list will be delivered to your inbox with ``[LRUG]`` in the subject line.  We suggest you set up mail rules or filters to keep them out of your inbox.
+
+The list exists to let the community stay in touch outside of the monthly meetings, but is open to all whether they have attended one of our meetings or not.  An average LRUG meeting attracts 100 or so people, but the list has over 1,000 members.  Be mindful of that fact when you send your messages: you're not just sending an email to a handful of friends.  To help everyone be courteous there are a handful of rules for the list:
+
+### Rules of the List
+
+New members of the mailing list will be on moderation for their first few posts to the list.
+
+Job ads are allowed on the list (see [below](#job-ads) for the rules), but the threads will be locked as soon as possible and replies deleted.
+
+Messages to the list that cause a complaint may be deleted and the thread they are a part of may be locked.
```

Comment:
Note that these two "rules" will require us to change the software that the list is provided by.  We currently use a mailman instance provided by the hosting provider (dreamhost) that the lrug.org domain is on.  We don't really have full control of the archives, and even if we did, we can't lock or moderated individual threads because it's not something that mailman does.  We're looking at moving to google groups or some other software solution, but we need to announce and plan that migration.